homeassistant/components/ezviz/alarm_control_panel.py

```diff
@@ -0,0 +1,162 @@
+"""Support for Ezviz alarm."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+from datetime import timedelta
+import logging
+
+from pyezviz import PyEzvizError
+from pyezviz.constants import DefenseModeType
+
+from homeassistant.components.alarm_control_panel import (
+    AlarmControlPanelEntity,
+    AlarmControlPanelEntityDescription,
+    AlarmControlPanelEntityFeature,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    STATE_ALARM_ARMED_AWAY,
+    STATE_ALARM_ARMED_NIGHT,
+    STATE_ALARM_DISARMED,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import (
+    DATA_COORDINATOR,
+    DOMAIN,
+    MANUFACTURER,
+)
+from .coordinator import EzvizDataUpdateCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+SCAN_INTERVAL = timedelta(seconds=60)
+PARALLEL_UPDATES = 0
+
+
+@dataclass
+class EzvizAlarmControlPanelEntityDescriptionMixin:
+    """Mixin values for EZVIZ Alarm control panel entities."""
+
+    ezviz_alarm_states: list
+
+
+@dataclass
+class EzvizAlarmControlPanelEntityDescription(
+    AlarmControlPanelEntityDescription, EzvizAlarmControlPanelEntityDescriptionMixin
+):
+    """Describe a EZVIZ Alarm control panel entity."""
+
+
+ALARM_TYPE = EzvizAlarmControlPanelEntityDescription(
+    key="ezviz_alarm",
+    translation_key="ezviz_alarm",
```

Comment:
You might need to test this on your device, but I think the states of Alarm Control Panel are already being translated by default, so you don't need an extra translation_key for this.

Reply:
Checked the strings under the Alarm_Control_Panel and it looks like you are correct. Also, can't find any integrations that implement translation on alarm_control_panel entities. (did a github search)

Removed it.